include/ginkgo/core/matrix/dense.hpp

```diff
@@ -407,6 +426,52 @@ class Dense : public EnableLinOp<Dense<ValueType>>,
                              values_.get_num_elems());
     }
 
+    /**
+     * Other implementations of dense should override this function instead of
```

Comment:
This is purely doxygen styling, but maybe for all of these you could use `@copydoc` with the appropriate function as source and put this sentence as a `@note`.

Reply:
Good idea, I will do that.

Reply:
That needs checking, but you also probably do not need to duplicate the `@param` comments when doing that.

Reply:
The current version works with the `@copydoc` (including the parameter).